xml/ha_maintenance.xml

```diff
@@ -0,0 +1,375 @@
+  </para>
+
+  <variablelist>
+   <varlistentry>
+    <term>Cluster Maintenance Mode</term>
+    <listitem>
+     <para>
+      Use this option if you need to perform maintenance tasks and do not want
+      to receive any notification by the cluster during that time. In
+      maintenance mode, you can stop or restart cluster resources at
+      will&mdash;the &hasi; will not attempt to restart them. All resources
+      automatically become unmanaged: The &hasi; will cease monitoring them and
+      thus be oblivious to their status.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Maintenance Mode For Resources</term>
+    <listitem>
+     <para>
+      Use this option if you need to manually touch a service that is managed
+      by a resource (or if you need to adjust any components of this service).
+     </para>
+     <note>
+      <title>Monitoring Unmanaged Resources</title>
+      <para>
+       If you want to suspend management of an individual resource but still
+       want the cluster to monitor the resource and to report any failures,
+       adjust the <option>is-managed</option> meta attribute for this resource
+       instead of putting the resource into maintenance mode. See <xref
+       linkend="pro.ha.config.hawk.rsc.is-managed" xrefstyle="select:label"/> or <xref
+       linkend="sec.ha.manual_config.cli.maint.mode" xrefstyle="select:label"/>
+       for details.
+      </para>
+      <para>
+       For the difference between maintenance mode and
+       <option>is-managed</option>, see also
+       <xref linkend="tab.ha.basics.meta"/>.
+      </para>
+     </note>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Maintenance Mode for Nodes</term>
+    <listitem>
+     <para>
+      Use this option if you want to update software packages on a node that do
+      <emphasis>not</emphasis> affect the cluster stack. To check the exact
+      conditions under which to use this option, see
+      <xref linkend="sec.ha.migration.update"/>.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+ </sect1>
+ <sect1 xml:id="se.ha.maintenance.cluster.services">
+  <title>Implications of Cluster Maintenance Mode</title>
+
+  <para>
+   If the cluster is in maintenance mode, all resources become unmanaged. This
+   means you can even stop all &pace; services on a node. All daemons and
+   processes (originally started as &pace;-managed cluster resources) will
+   continue to run.
+  </para>
+
+  <para>
+   However, as &corosync; is also stopped when you stop the &pace; service,
+   this might lead to unexpected results for some resources. For example, DLM
+   depends on the membership and messaging services provided by &corosync;. If
+   &corosync; stops, the DLM resource will assume a split-brain scenario and
+   trigger a fencing operation. To avoid this, manually stop resources like DLM
+   or bring them back from maintenance mode before stopping the &pace; service.
```

Comment:
To avoid this, explicitly stop resources like DLM before putting cluster or node into maintenance mode and stopping the &pace; service.

Reply:
Changed, thanks.